nap/docker/ckan/ckan-plugins/ckanext-napote_theme/ckanext/napote_theme/i18n/fi/LC_MESSAGES/ckanext-napote_theme.po

```diff
@@ -0,0 +1,313 @@
+# Finnish translations for ckanext-napote_theme.
+# Copyright (C) 2017 ORGANIZATION
+# This file is distributed under the same license as the
+# ckanext-napote_theme
+# project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2017.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: ckanext-napote_theme 1.0.0\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2017-11-03 12:49+0000\n"
+"PO-Revision-Date: 2017-11-03 16:14+0200\n"
+"Language: fi\n"
+"Language-Team: fi <LL@li.org>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.3.4\n"
+"Last-Translator: \n"
+"X-Generator: Poedit 1.8.7.1\n"
+
+#: ckanext/napote_theme/plugin.py:116
+#: ckanext/napote_theme/templates/header.html:36
+msgid "Organizations"
+msgstr "Palveluntuottajat"
+
+#: ckanext/napote_theme/plugin.py:117 ckanext/napote_theme/plugin.py:128
+#: ckanext/napote_theme/templates/package/read.html:9
+msgid "Transport Service Type"
+msgstr "Liikkumispalvelun tyyppi"
+
+#: ckanext/napote_theme/plugin.py:118 ckanext/napote_theme/plugin.py:129
+#: ckanext/napote_theme/templates/package/snippets/package_basic_fields.html:11
+msgid "Operation Area"
+msgstr "Toiminta-alue"
+
+#: ckanext/napote_theme/plugin.py:119 ckanext/napote_theme/plugin.py:130
+msgid "Tags"
+msgstr ""
+
+#: ckanext/napote_theme/plugin.py:120 ckanext/napote_theme/plugin.py:131
+msgid "Formats"
+msgstr ""
+
+#: ckanext/napote_theme/plugin.py:121 ckanext/napote_theme/plugin.py:132
+msgid "Licenses"
+msgstr ""
+
+#: ckanext/napote_theme/plugin.py:191
+msgid "Transport Service View"
+msgstr "Liikkumispalvelunäkymä"
+
+#: ckanext/napote_theme/templates/base.html:3
+msgid "Data and Resources"
+msgstr "Rajapinnat"
```

Comment:
This could be something like "Koneluettavat palvelut" ? We have a meeting today where we can talk this through.

Reply:
Nope. You are already in the "service/palvelu"-page when viewing the APIs provided by the service.